src/tests/conformance/query/get_by_hash.rs

```diff
@@ -30,13 +30,10 @@ async fn c007_TM_GET_OBJECT_BY_HASH_get_transaction_by_hash() {
 
     // Wait for correct state and account data.
     wait_for_state(&node.rpc_url(), "proposing".into()).await;
-    let account_data = wait_for_account_data(
-        &node.rpc_url(),
-        "rNGknFCRBZguXcPqC63k6xTZnonSe6ZuWt",
-        EXPECTED_RESULT_TIMEOUT,
-    ) // TODO make constant
-    .await
-    .expect("unable to get account data");
+    let account_data =
+        wait_for_account_data(&node.rpc_url(), TEST_ACCOUNT, EXPECTED_RESULT_TIMEOUT) // TODO make constant
```

Comment:
Maybe we can remove the TODO comment now?

Reply:
Done